include/eosio/vm/execution_context.hpp

```diff
@@ -204,6 +224,7 @@ namespace eosio { namespace vm {
       std::error_code                 _error_code;
       operand_stack                   _os;
       std::vector<init_expr>          _globals;
+      bool                            _is_jit = false;
```

Comment:
I don't think it matters but this could be a template parameter instead I think.